integration-tests/go-sql-server-driver/testdef.go

```diff
@@ -323,12 +487,22 @@ func RunQueryAttempt(t require.TestingT, conn *sql.Conn, q driver.Query) {
 		rowstrings, err := RowsToStrings(len(cols), rows)
 		require.NoError(t, err)
 		if q.Result.Rows.Or != nil {
-			require.Contains(t, *q.Result.Rows.Or, rowstrings)
+			match := *q.Result.Rows.Or
```

Comment:
What's this about? Why would we need the dynamic ports to check the result?

Reply:
There are some tests which assert on the output of a call to `dolt_cluster_transition_to_standby`, and that call has a schema where the full remote url is in the response. For example:

```yaml
    - query: "call dolt_cluster_transition_to_standby('2', '1')"
      result:
        columns: ["caught_up", "database", "remote", "remote_url"]
        rows:
        - ["1", "repo1", "standby", "http://localhost:{{get_port \"server2_cluster\"}}/repo1"]
        - ["1", "mysql", "standby", "http://localhost:{{get_port \"server2_cluster\"}}"]
        - ["1", "dolt_branch_control", "standby", "http://localhost:{{get_port \"server2_cluster\"}}"]
```

This seemed like an OK approach...